src/compiler/crystal/codegen/llvm_builder_helper.cr

```diff
@@ -111,11 +113,11 @@ module Crystal
       builder.inbounds_gep type, ptr, index0, index1, name
     end
 
-    def call(func, name : String = "")
+    def call(func : LLVM::Function, name : String = "")
```

Comment:
Is it correct to understand that this path isn't changed, meaning, current `call` calls that do not provide a `LLVMTypedFunction` won't use the typed call?  Is there a reason to keep it? (Same with invoke)

Reply:
There are actually no more current calls that use these overloads, I think I just forgot to remove them to maintain parity with the "public" LLVM bindings (but that's unnecessary because the helper module here isn't public)